src/foam/box/SessionServerBox.java

```diff
@@ -124,7 +124,7 @@ public void send(Message msg) {
 
       msg.getLocalAttributes().put("x", effectiveContext);
     } catch (Throwable t) {
-      logger.error("Error throw in SessionServerBox: " + t, " ,service: " + spec.getName());
+      logger.error("Error throw in SessionServerBox: " + t, " ,service: " + spec.getName(), t);
```

Comment:
Isn't `t` already included in the log earlier in this line?

Reply:
Suspect the + t performs a + t.toString() . rather then the intended    , t ,   which would have generated the stack trace.